providers/mailersend/src/lib/mailersend.provider.ts

```diff
@@ -22,24 +22,26 @@ export class MailersendEmailProvider implements IEmailProvider {
     this.mailerSend = new MailerSend({ api_key: this.config.apiKey });
   }
 
-  async sendMessage(
-    options: IEmailOptions
-  ): Promise<ISendMessageSuccessResponse> {
-    let recipients;
-    if (Array.isArray(options.to)) {
-      recipients = options.to.map((recipient) => {
-        return new Recipient(recipient);
-      });
-    } else {
-      recipients = new Recipient(options.to);
-    }
+  private createRecipients(
+    recipients: IEmailOptions['to']
+  ): Recipient[] | Recipient {
```

Comment:
```suggestion
  private createRecipients(
    recipients: IEmailOptions['to']
  ): Recipient[] {
```
We are returning an array always.

Reply:
Wow, good you noticed it!